grpc/py-cubic/setup.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name='cobalt-cubic',
+    python_requires='>=3.5.0',
+    version='2.0.0',
```

Comment:
This version should be set to the same value as the [go sdk](https://github.com/cobaltspeech/sdk-cubic/blob/master/grpc/go-cubic/cubicpb/gw/go.mod#L6) and the [README](https://github.com/cobaltspeech/sdk-cubic/blob/master/README.md#tagging-new-versions) needs to be changed to also update this value before tagging a release.